packages/twilio-worker-auth/src/user.ts

```diff
@@ -0,0 +1,10 @@
+export type User = {
+  workerSid: string;
+
+  roles: string[];
+};
+
+export const user = (workerSid: string, roles: string[]): User => ({
```

Comment:
See [comment from above](https://github.com/techmatters/hrm/pull/284/files#diff-5f384ea3d60310f21ed7811f4a2fa3d4941147ef359d95460e942281ea1cf7d1R7) that really probably belongs here.